scripts/vscripts/tf2ware_ultimate/botbehavior/minigames/basketball.nut

```diff
@@ -0,0 +1,122 @@
+
+// Constants
+const GRENADE_SPEED = 1216.6;    // hammer units/s
```

Comment:
Do not use const, use local instead. Pass these as parameters to the function below